imi/src/main/java/org/motechproject/nms/imi/service/impl/CsvHelper.java

```diff
@@ -130,8 +130,13 @@ public static CallDetailRecordDto csvLineToCdrDto(String line) {
 
         cdr.setMsisdn(msisdnFromString(fields[FieldName.MSISDN.ordinal()]));
 
-        cdr.setCallAnswerTime(new DateTime(longFromString("CallAnswerTime",
-                fields[FieldName.CALL_ANSWER_TIME.ordinal()])));
+        try {
+            cdr.setCallAnswerTime(new DateTime(longFromString("CallAnswerTime",
```

Comment:
This will throw IllegalArgumentException if the number is malformed and if it's null. How about making something like `longOrNullFromString()`?